src/node_win32_etw_provider-inl.h

```diff
@@ -253,7 +247,11 @@ void NODE_V8SYMBOL_ADD(LPCSTR symbol,
     }
     void* context = nullptr;
     INT64 size = (INT64)len;
-    INT_PTR id = (INT_PTR)addr1;
+#if defined(_WIN64)
+    INT32 id = 0;
```

Comment:
Why do we not just use the lower 32 bits on 64 bit? That is, why not just have the non-64 bit version on 64 bit too? 

Also agree with @refack that using addr1 as the id is somewhat arbitrary but I don't know if v8 has a better (and more stable/deterministic) id that we can use here- in ChakraCore we use our "function id" which I believe is somewhat stable across runs of node because it's incremented in order of in which the function is compiled- this allows us to collect a trace, and then run with additional tracing flags for just that function id to get more details on that function. Does v8 have something stable like that that we can use here? (cc @fhinkel)